packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2610,12 +2610,11 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     final bool newTickerEnabled = TickerMode.of(context);
     if (_tickersEnabled != newTickerEnabled) {
       _tickersEnabled = newTickerEnabled;
-      if (_tickersEnabled && _cursorActive) {
+      final bool needsBlinkingCursor = _hasFocus && _value.selection.isCollapsed && widget.showCursor;
```

Comment:
Consider moving this into a getter also why not move _tickersEnabled into this bool like the one below does